src/main/java/com/tg5/service/RoleServiceImpl.java

```diff
@@ -22,7 +22,9 @@ public class RoleServiceImpl extends BaseReadWriteServiceImpl<RolePayload, Role,
 
     @Override
     public RolePayload create(RolePayload request) {
```

Comment:
Are you sure we can't do without this? And if we do create, do we need udpate as well?